recipes/ml-collections/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "ml-collections" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ml_collections-{{ version }}.tar.gz
+  sha256: 3fefcc72ec433aa1e5d32307a3e474bbb67f405be814ea52a2166bfc9dbe68cc
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - absl-py
+    - pyyaml
+    - six
+    - contextlib2
+    - dataclasses  # [py<37]
```

Comment:
conda-forge does not build for python versions <3.7 (and that's going to be dropped in the near future https://conda-forge.org/docs/user/announcements.html#dropping-python-3-7), so I think you can make this `noarch: python` and then require `python >= 3.7`, dropping the `dataclasses` and `typing` requirements.

Reply:
Good idea. Will do.